axlearn/common/attention.py

```diff
@@ -122,6 +121,9 @@
     maybe_instantiate,
 )
 from axlearn.common.flash_attention.remat import FLASH_ATTN_RESIDUAL_NAME
+from axlearn.common.kv_cache.base_kv_cache import BaseKVCache, KVState
+from axlearn.common.kv_cache.kv_cache import KVCache
+from axlearn.common.kv_cache.sliding_window_kv_cache import SlidingWindowKVCache
```

Comment:
Not blocking this PR, but over the long run we'd like to remove this dependency.

Reply:
Done.